kobo/apps/openrosa/apps/logger/migrations/0044_attachment_uid.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.15 on 2025-03-11 17:18
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('logger', '0043_attachment_date_created_attachment_date_modified'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='attachment',
+            name='uid',
+            field=models.CharField(blank=True, db_index=True, max_length=23, null=True),
```

Comment:
If we are going to use a prefix with 3 letters (like `att`) , the max_length should 24. 
Ensure to match the length accordingly.